src/binary_cache.cpp

```diff
@@ -98,8 +94,9 @@ KDb GetDb(const std::string& device, size_t num_cu)
     static const auto user_dir = ComputeUserCachePath();
     static const auto sys_dir  = ComputeSysCachePath();
     boost::filesystem::path user_path =
-        user_dir / (Handle::GetDbBasename(device, num_cu) + ".ukdb");
-    boost::filesystem::path sys_path = sys_dir / (Handle::GetDbBasename(device, num_cu) + ".kdb");
+        user_dir / (Handle::GetDbBasename(device, num_cu) + "_" MIOPEN_SQLITE_SCHEMA_VER ".ukdb");
+    boost::filesystem::path sys_path =
+        sys_dir / (Handle::GetDbBasename(device, num_cu) + "_" MIOPEN_SQLITE_SCHEMA_VER ".kdb");
```

Comment:
The schema version is for the perfdb not the binary cache.

Reply:
I think a separate version number for each database is excessive, particularly since these may be merged into one

Reply:
What do you mean?

Reply:
While, currently we have a separate database for perf db, kernel cache and (in the future) find dd. They are essentially tables which may be part of the same `db` file. Thus may be merged.

Reply:
I see what you meant.

There is only one schema version for perfdb. The finddb and binary cache will use the miopen version.

Finddb already writes the miopen version in the name so it just needs to include the tweak version.

The binary cache already writes a separate subdirectory for each version so it just needs to be extended to include the tweak version.

Reply:
Fixed